sync-core/src/main/java/com/cloudant/sync/query/QuerySqlTranslator.java

```diff
@@ -468,6 +498,27 @@ protected static SqlParts whereSqlForAndClause(List<Object> clause, String index
         return SqlParts.partsForSql(where, parameterArray);
     }
 
+    private static String buildWhereClauseArgument(List<Object> values,
```

Comment:
Changed `buildWhereClauseArgument` to `placeholdersForInList` as the method has become more clear/concise due to the removal of redundant predicate validation.  Done in 5b5040e6a64ad6720ef17ace362f2020397d0880.